.github/workflows/build.yml

```diff
@@ -152,13 +193,13 @@ jobs:
       - name: build discourse_test:slim
         run: |
           docker buildx build . --load \
-            --build-arg from_tag=slim \
```

Comment:
The `slim` tag for `discourse/base` no longer exists. We need to be using `release-slim`.

Reply:
This was a mistake when I refactored the build recently for multiarch support. Shall we go back to calling it `:slim`? Or do you prefer `release-slim`? (I don't mind either way 🤷‍♂️)

Reply:
Maybe slim just for consistency sake since that has always been the case?

Reply:
💯